docker/dev/docker-compose.yml

```diff
@@ -100,6 +100,7 @@ services:
     ports:
       - '3006:3006'
       - '3008:3008'
+      - '3009:3009'
```

Comment:
it is not necessary to expose the port as the request is from wallet-backend to rafiki-auth in the docker network